app/models/javascript_cache.rb

```diff
@@ -8,7 +8,7 @@ class JavascriptCache < ActiveRecord::Base
   before_save :update_digest
 
   def url
-    "#{GlobalSetting.cdn_url}#{GlobalSetting.relative_url_root}/theme-javascripts/#{digest}.js?__ws=#{Discourse.current_hostname}"
+    "#{GlobalSetting.cdn_url}#{Discourse.base_uri}/theme-javascripts/#{digest}.js?__ws=#{Discourse.current_hostname}"
```

Comment:
Does this make a difference? I'm pretty sure these two things evaluate to the same result. This is from the console on a live subfolder site

```
[1] pry(main)> GlobalSetting.relative_url_root
=> "/forum"
[2] pry(main)> Discourse.base_uri
=> "/forum"
```

Reply:
So... maybe this is already working. And all we need to do it change the CSP :)

Switching to use `base_uri` is a good change anyway, so let's leave it in 👍

Reply:
The site I diagnosed this on is definitely getting an empty string back for this.